prompt.py

```diff
@@ -437,6 +437,8 @@ def show_wallet(self, arguments):
                 AddAlias(self.Wallet, arguments[1], arguments[2])
             else:
                 print("Please supply an address and title")
+        else:
```

Comment:
did you merge development into you branch before pushing this PR? i think that's already in there

Reply:
I committed my local changes, merged coz/development/ into my local branch, pushed to my own remote, PR'ed from there to here. I don't have control over keeping `coz/feature-consensus` up to date with `coz/development` and @localhuman said on slack "make sure to merge early and often from development" so that's what I did?